components/automate-cs-ocid/habitat/plan.sh

```diff
@@ -63,8 +65,8 @@ do_install() {
   bundle install
 
   # This is required so that 'db:migrate' can be executed from the run hook
-  chmod 766 -R "$(hab pkg path 'chef/oc_id')/oc_id/db/schema.rb"
-  chmod 777 -R "$(hab pkg path 'chef/oc_id')/oc_id/config"
+  chmod 766 -R "db/schema.rb"
+  chmod 777 -R "config"
```

Comment:
Check if we can manage with change of permission only on specific files/folders without setting permission on the whole `config` folder